og.module

```diff
@@ -166,7 +166,7 @@ function og_entity_access(EntityInterface $entity, $operation, AccountInterface
     return AccessResult::forbiddenIf($node_access_strict);
   }
 
-  return AccessResult::forbidden();
+  return AccessResult::neutral();
```

Comment:
Maybe we should adapt the above `$node_access_strict = \Drupal::config('og.settings')->get('node_access_strict');` to be a general `$entity_access_strict = \Drupal::config('og.settings')->get('entity_access_strict');`